docs/repositories-configure/adjusting-quality-settings.md

```diff
@@ -21,7 +21,14 @@ These settings configure when Codacy reports pull requests and commits as not up
 -   **New security issues are over:** Pull requests or commits are marked not up to standards if the number of security issues introduced is higher than the set value. By default, this setting is off.
 -   **Complexity is over:** Pull requests or commits are marked not up to standards if the introduced complexity is higher than the set value. By default, this setting is off.
 -   **Duplication is over:** Pull requests or commits are marked not up to standards if the number of clones introduced is higher than the set value. By default, this setting is off.
--   **Coverage variation is under:** Pull requests or commits are marked not up to standards if the coverage introduced is smaller than the set value. This value refers to the variation of coverage on your pull requests or commits. For example, if your repository has 90% coverage and you want to make sure that this value doesn't decrease, set the value to 0%. By default, this setting is off. The maximum value is 1%.
+-   **Coverage variation is under:** Pull requests or commits are marked not up to standards if they introduce a variation to coverage below the set value. The maximum value is 1%. By default, this setting is off.
+
+    !!! tip
+        To ensure that commits and pull requests:
+
+        -   **Can decrease** the coverage, set the value to a negative percentage
```

Comment:
@prcr @fjrdomingues Just to confirm here, I was under the impression that is not possible to set more that -1%, meaning we cannot say -3% etc. Is that true?

Reply:
The minimum value is -100% and the maximum is 1%:

![image](https://user-images.githubusercontent.com/60105800/152316782-e85713c0-a6f3-4f62-be92-849c028efd25.png)

You can also check here for more context:

* https://codacy.atlassian.net/browse/CY-4370
* https://codacy.atlassian.net/browse/CY-5374?focusedCommentId=48607

Reply:
Thank you